tests/classifier.ini

```diff
@@ -0,0 +1,80 @@
+;; small classification test
+
+[main]
+name="classification"
+tf_manager=<tf_manager>
+output="tests/tmp-test-output"
+overwrite_output_dir=True
+batch_size=16
+epochs=4
+train_dataset=<train_data>
+val_dataset=<val_data>
+trainer=<trainer>
+runners=[<runner>]
+postprocess=None
+evaluation=[("classification", <accuracy>)]
+logging_period=50
+validation_period=100
+runners_batch_size=1
+random_seed=1234
+
+[tf_manager]
+class=tf_manager.TensorFlowManager
+num_threads=4
+num_sessions=1
+
+[accuracy]
+class=evaluators.accuracy.Accuracy
+
+[train_data]
+class=dataset.load_dataset_from_files
+s_source="tests/data/train.tc.en"
+s_classification="tests/data/train.words"
+preprocessors=[("source", "source_chars", processors.helpers.preprocess_char_based)]
+lazy=True
+
+[val_data]
+class=dataset.load_dataset_from_files
+s_source="tests/data/val.tc.en"
+s_classification="tests/data/val.words"
+preprocessors=[("source", "source_chars", processors.helpers.preprocess_char_based)]
+
+[encoder_vocabulary]
+class=vocabulary.from_file
+path="tests/tmp-vocab/encoder_vocab.pkl"
```

Comment:
jsi si jistej timhle? Bude to slovník, kterej se vytvořil ve `vocab.ini` a bude to tim pádem potřeba vždy pouštět až pozdějc.